src/linagora.esn.unifiedinbox/views/thread/list/list-item.pug

```diff
@@ -38,7 +38,9 @@
             .indicators.esn-flex-noshrink.visible-xl
               ul.actions.flex-vertical-centered
                 li.action-item
-                  i.inbox-message-indicators(dynamic-directive='inbox-message-indicators')
+                  i.inbox-message-indicators
+                    cal-inbox-invitation-message-indicator(ng-if="ctrl.shouldDisplayCalendarInvitationMessageIndicator")
+                    cal-inbox-resource-management-indicator(ng-if="ctrl.shouldDisplayCalendarResourceManagementIndicator")
```

Comment:
So when `ctrl.shouldDisplayCalendarInvitationMessageIndicator = false` and `ctrl.shouldDisplayCalendarResourceManagementIndicator = false`, only an empty `i.inbox-message-indicators` will be left right? Will it occupy some space and result in some weird UI errors?

Reply:
It works without UI errors before this PR